src/test/e2e/api/openapi/openapi.e2e.test.ts

```diff
@@ -100,3 +101,91 @@ test('the generated OpenAPI spec is valid', async () => {
 
     expect(enforcerWarning ?? enforcerError).toBe(undefined);
 });
+
+test('all root-level tags are "approved tags"', async () => {
+    const { body: spec } = await app.request
+        .get('/docs/openapi.json')
+        .expect('Content-Type', /json/)
+        .expect(200);
+
+    const specTags = spec.tags;
+    const approvedTags = openApiTags;
+
+    expect(specTags).toStrictEqual(approvedTags);
```

Comment:
I wonder if instead of strictly checking this we should check that all specTags are included in approvedTags. Despite, it's good to have this test for housekeeping reasons

Reply:
Actually, that's a very good point! Checking that specTags is a subset of approvedTags is probably better 🧠  I'll fix!